### ISSUE TITLE
Add unit tests for the device manager functionality

### ISSUE THREAD

**Issue by manojVivek** [responsively-org/responsively-app#934]

**Comment by horrificCoder** [comment]
@manojVivek Can I work on this?

**Comment by manojVivek** [comment]
@horrificCoder That would be fantastic!

**Comment by ReshadSadik** [comment]
Hi, @horrificCoder  are you working on this ? If not I would love to pick this up . @manojVivek

**Comment by elidakirigo** [comment]
hi, Is anyone working on the unit tests? @manojVivek i'd love to contribute

**Comment by manojVivek** [comment]
@elidakirigo Yes, that would be fantastic, please go ahead and let us know if you have any questions.

**Comment by elidakirigo** [comment]
> @elidakirigo Yes, that would be fantastic, please go ahead and let us know if you have any questions.

thanks
